packages/gtk3.rb

```diff
@@ -49,26 +51,33 @@ def self.build
             builddir"
     system 'meson configure builddir'
     system 'ninja -C builddir'
-    @file = <<~GTK3_CONFIG_HEREDOC
-      [Settings]
-      gtk-application-prefer-dark-theme = false
-      gtk-icon-theme-name = hicolor
-      gtk-fallback-icon-theme = gnome
-      gtk-font-name = Arial 10
-    GTK3_CONFIG_HEREDOC
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
     system "sed -i 's,null,,g'  #{CREW_DEST_LIB_PREFIX}/pkgconfig/gtk*.pc"
     FileUtils.mkdir_p("#{CREW_DEST_HOME}/.config/gtk-3.0")
-    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @file)
+    File.write("#{CREW_DEST_HOME}/.config/gtk-3.0/settings.ini", @gtk3settings)
   end
 
   def self.postinstall
     # generate schemas
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
     # update mime database
     system "update-mime-database #{CREW_PREFIX}/share/mime"
+    @xdg_config_home = ENV['XDG_CONFIG_HOME']
+    @gtk3settings = <<~GTK3_CONFIG_HEREDOC
+      [Settings]
+      gtk-application-prefer-dark-theme = false
+      gtk-icon-theme-name = hicolor
+      gtk-fallback-icon-theme = gnome
+      gtk-font-name = Arial 10
+    GTK3_CONFIG_HEREDOC
```

Comment:
Move to `def self.build` instead?

Reply:
The heredoc isn't actually being written to disk unless the condition in postinstall is true.  (This way you can run `crew postinstall gtk3`)

Reply:
What about creating `settings.ini.example` and include it in the package files but then copy `settings.ini.example` to `settings.ini` if it doesn't exist?  If we do this then we will want a `def self.remove` section to optionally remove `settings.ini` when the package is removed.

Reply:
I think there's plenty of documentation on settings for gtk out there. By just including a default settings.ini file, we bypass all this craziness. People are free to edit their local `gtk-{3,4}/settings.ini` as they see fit